releasenotes/notes/update-instruction-alignment-passes-ef0f20d4f89f95f3.yaml

```diff
@@ -1,31 +1,165 @@
-    and :class:`InstructionDurationCheck`. The new pass :class:`ConstrainedReschedule`
-    considers both hardware alignment constraints of ``pulse_alignment`` and ``acquire_alignment``,
-    which is a drop-in replacement of :class:`AlignMeasures`.
-upgrade:
+    Added a new transpiler pass :class:`~.ConstrainedReschedule` pass.
+    The :class:`ConstrainedReschedule` pass considers both hardware
+    alignment constraints that can be definied in a :class:`.BackendConfiguration`
+    object, ``pulse_alignment`` and ``acquire_alignment``. This new class superscedes
+    the previosuly existing :class:`~.AlignMeasures` as it performs the same alignment
+    (via the property set) for measurement instructions in addition to general instruction
+    alignment. By setting the ``acquire_alignment`` constraint for
+    :class:`ConstrainedReschedule` which is a drop-in replacement of
+    :class:`~.AlignMeasures` when paired with a new :class:`~.BasePadding` pass.
   - |
-    The ordering of scheduling-relevant passes in the preset pass manager has been upgraded.
-    Note that now scheduling passes and alignment passes are all sub-class of the
-    :class:`AnalysisPass` that only updates the property set of the pass manager,
-    in which new property set ``node_start_time`` is created there to perform
-    these operation on nodes scheduled on the absolute time.
-
-    Previously, the pass chain had been implemented as ``scheduling -> alignment``
-    which were both transform passes thus there were multiple DAG instance regeneration.
-    In addition, scheduling occured in each pass to obtain instruction start time.
-
-    Now this chain becomes ``scheduling -> alignment -> padding``
-    where the DAG replacement only occurs at the end, i.e. at the ``padding`` pass.
-    Note that who wants to design a new pass manager must insert one of the padding passes
-    at the end of the pass chain. Without the padding pass, the scheduled nodes
-    are not reflected in the target code since the QASM payload has no notion of
-    the scheduling for underlying instructions.
-deprecations:
+    Added new transpiler passes :class:`~.ALAPScheduleAnalysis` and :class:`~.ASAPScheduleAnalysis`
+    which superscede the :class:`~.ALAPSchedule` and :class:`~.ASAPSchedule` as part of the
+    reworked transpiler workflow for schedling. The new passes perform the same scheduling but
+    in the property set and relying on a :class:`~.BasePadding` pass to adjust the circuit
+    based on all the scheduling alignment analysis.
+
+    These new passes also consider I/O latencies for measurement and conditional operations.
+    These information should be set by :class:`SetIOLatency` pass in advance.
+    The I/O latency to write classical register is represneted by ``clbit_write_latency``,
+    and one for reading the register is presented by ``conditional_latency``.
+    These properties are loaded from backend and set to the pass manager's property set.
```

Comment:
This is not implemented because we didn't define the spec for these values. `SetIOLatency` is not a part of preset pass managers now because of this.

Reply:
Sorry then that was my mistake. Please drop this line.

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/7835/commits/bae9f47aaa3ef990884f3de4a23ad439a0f59c52